timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -1464,7 +1474,7 @@ def select_target(ts_df: TimeSeriesDataFrame) -> TimeSeriesDataFrame:
             target=self.target,
             prediction_length=self.prediction_length,
             eval_metric=self.eval_metric.name,
-            eval_metric_seasonal_period=self.eval_metric_seasonal_period,
+            eval_metric_seasonal_period=self.eval_metric.seasonal_period,
```

Comment:
why do we need `seasonal_period` in simulation artifacts? for bookkeeping and consistency only? if so, should horizon weight also be here?

Reply:
Good point